### ISSUE TITLE
Missing the hotkey label for the 10th entry.

### ISSUE THREAD

**Issue by trmdi** [psifidotos/Latte-Dock#816]
Look at this image:
![image](https://user-images.githubusercontent.com/11857205/34398642-116f1a28-ebb3-11e7-9cc0-c28cb4f32542.png)
Btw, I think these label should be labeled 1234567890 like the keys on the keyboard. (use the name "0" instead of "10")

**Comment by psifidotos** [comment]
Unity is doing something different, 10=a, 11=b, 12=c .....

**Comment by trmdi** [comment]
Yeah, that is something I've thought about, could you also do that?
